csrc/type.h

```diff
@@ -336,29 +342,16 @@ struct IsPrimitiveNativeType : std::false_type {};
   template <>                                                  \
   struct IsPrimitiveNativeType<native_type> : std::true_type {}
 
-#define DEFINE_DATATYPE_TO_ATEN_AND_NATIVE_TYPE(                 \
-    data_type, at_type, native_type)                             \
-  template <>                                                    \
-  struct DataTypeToNativeType<data_type> {                       \
-    using type = native_type;                                    \
-  };                                                             \
```

Comment:
Was this unused? Could we use this in the switch statement below in getPrimDataTypeSize?

Reply:
They were used. They were just a copy-paste of `DEFINE_DATATYPE_TO_NATIVE_TYPE`, so I replaced the copy-pasted code with `DEFINE_DATATYPE_TO_NATIVE_TYPE`.

Reply:
This can not be used in `primDataTypeSize` either, because this requires the data type to be compile-time constant, which is not the case for `primDataTypeSize`.